RELOAD-273, RELOAD-274, RELOAD 275

bravewick opened the pull request:
Fixing bugs, code cleanup and tweaks

Commit:
RELOAD-298 Refactoring of the classes, layouts, orientations, etc

Commit:
Merge remote-tracking branch 'upstream/OneOne' into OneOne

Commit:
RELOAD-298 Fix the mosync image name in resource files list

Commit:
RELOAD-298 Corrected the font color in stored project lsit screen for IOS and fixed the tab icons to be completely transparent

TechWizEric commented on ReloadClient/MainScreen.h:
```diff
@@ -40,7 +40,7 @@ class MainScreen :
 	/**
 	 * Called just before the screen begins rotating.
```
Comment no longer applies.

TechWizEric commented on ReloadClient/LoginLayout.cpp:
```diff
@@ -0,0 +1,367 @@
+/*
+ * LoginLayout.cpp
+ *
+ *  Created on: Feb 4, 2013
```
Date and Author may be incorrect.

TechWizEric commented on ReloadClient/LoginLayout.h:
```diff
@@ -17,22 +17,27 @@
  *
- *  Created on: Mar 25, 2013
- *      Author: Kostas Tsolakis
+ *  Created on: Feb 4, 2013
```
Date and Author may be incorrect.

TechWizEric commented on ReloadClient/MainScreen.cpp:
```diff
@@ -24,12 +24,13 @@
 MainScreen::MainScreen():
 	TabScreen()
 {
    // do init stuff
```
Refine comment.

TechWizEric commented on ReloadClient/MainScreen.cpp:
```diff
@@ -40,12 +41,12 @@
 /**
  * Called just before the screen begins rotating.
```
Comment no longer applies.

TechWizEric commented on ReloadClient/ReloadClient.cpp:
```diff
@@ -161,18 +162,24 @@ static void DeleteFolderRecursively(const char *path)
 {
-	lprintfln("@@@ ReloadClient");
 	// Initialize application.
 	// Order of calls are important as data needed by
```
Comment may not still apply.

Commit:
RELOAD-298 Fix Author, and remove not needed code. Fix
the reloading to all clients issue when reloading a project from a particular
device.

TechWizEric commented:
Code review OK.

Pull request also encompasses RELOAD-280.
